Source/Scene/Model.js

```diff
@@ -3462,12 +3461,16 @@ define([
                 // GLTF_SPEC: Offical means to determine translucency. https://github.com/KhronosGroup/glTF/issues/105
                 var isTranslucent = rs.blending.enabled;
                 var owner = {
-                    primitive : defaultValue(model.pickPrimitive, model),
+                    primitive : defaultValue(model._pickObject, model),
                     id : model.id,
                     node : runtimeNode.publicNode,
                     mesh : runtimeMeshesByName[mesh.name]
                 };
 
+                if (defined(model._pickObject)) {
+                    owner = model._pickObject;
+                }
```

Comment:
The logic here can be reversed to avoid the `owner` allocation. Something like

```
var owner = model._pickObject;
if (!defined(owner)) {
    owner = ...
}
```